server/auth/types/jwt/jwt_auth.ts

```diff
@@ -25,10 +25,21 @@ import {
   AuthToolkit,
   IOpenSearchDashboardsResponse,
 } from 'opensearch-dashboards/server';
+import { ServerStateCookieOptions } from '@hapi/hapi';
 import { SecurityPluginConfigType } from '../../..';
 import { SecuritySessionCookie } from '../../../session/security_cookie';
 import { AuthenticationType } from '../authentication_type';
 import { JwtAuthRoutes } from './routes';
+import {
+  ExtraAuthStorageOptions,
+  getExtraAuthStorageValue,
+  setExtraAuthStorage,
+} from '../../../session/cookie_splitter';
+
+export const JWT_DEFAULT_EXTRA_STORAGE_OPTIONS: ExtraAuthStorageOptions = {
```

Comment:
I added this const here because of the JWT config schema being defined as `maybe`.
So this means that if no config for JWT is present in `opensearch_dashboards.yml`, there won't be any JWT values available in the config passed to the plugin, despite default values being used.

So that means that a customer may have JWT defined as authentication type, but unless they've changed any JWT config values, the plugin won't have access to the (default) config.
For SAML and OIDC we didn't want to make the cookie splitting configurable, so I went the same path here. And to make sure we don't run into problems when setting up the cookies, I used this constant as a fallback for the default values. You can see the usage on line 92 in this file.

I think we could just remove the `maybe` from the JWT config schema, but I didn't want to change too much without knowing the reason for it being there in the first place.